sharing/metadata/index.md

```diff
@@ -0,0 +1,57 @@
+---
+author:
+  display_name: AGRC Office
+  email: agrc@utah.gov
+tags:
+  - metadata
+  - sgid
+date: 2020-07-09 09:33:34 -0600
```

Comment:
**chore**: update the date if this is a new page

Reply:
What's the proper process for the date tag? I was under the impression Jekyll automatically updated it when it builds the site.

Reply:
Nope just takes it from the Metadata or the filename. Either way there is no magic.

Reply:
👍